openhub/management/commands/import_organization_data.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand
+from openhub.organization import get_organization_data, import_data
+
+
+class Command(BaseCommand):
+    help = 'Import Organization data'
+
+    ORG = get_organization_data()
```

Comment:
instantiating a class shouldnt do real work.  it should only do set up.

in this case, `get_organization_data` has all sorts of exceptions which could occur and wont be managed easily